tests/projects/utils.py

```diff
@@ -46,7 +48,9 @@ def build_docker_example_base_image():
 @pytest.fixture()
 def tracking_uri_mock(tmpdir):
     try:
-        mlflow.set_tracking_uri(os.path.join(tmpdir.strpath, 'mlruns'))
+        path = os.path.abspath(os.path.join(tmpdir.strpath, 'mlruns'))
```

Comment:
did you mean to undo these projects changes, or do they not matter anyway?

Reply:
I do want to revert these, good catch.